src/gatsby/models/json-files/index.js

```diff
@@ -1,13 +1,28 @@
 const callOnModels = require('../../utils/models')
+const yaml = require('js-yaml')
+
+async function parseContentToJson({ node, loadNodeContent }) {
+  switch (node.extension) {
+    case 'json':
+      return JSON.parse(await loadNodeContent(node))
+
+    case 'yml':
```

Comment:
probably we don't need this for now - let's simplify the logic back?

Reply:
Absolutely! I forgot about that bit when removing the rest.